keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/verification/VerificationRepository.kt

```diff
@@ -27,6 +27,14 @@ interface VerificationRepository {
   )
 
   fun nextEnvironmentsForVerification(minTimeSinceLastCheck: Duration, limit: Int) : Collection<VerificationContext>
+
+  fun wasSuccessfullyVerifiedIn(
+    deliveryConfig: DeliveryConfig,
+    artifactReference: String,
+    version: String,
+    environmentName: String,
+    verification : Verification
+  ): Boolean
```

Comment:
Is this actually implemented somewhere?

Reply:
I mean it must be implemented somewhere, so that this compiles... :D

Reply:
Only if there is an implementation of the interface...

Reply:
Yeah, the method is implemented in SqlVerificationRepository.